version.php

```diff
@@ -26,5 +26,6 @@
 defined('MOODLE_INTERNAL') || die();
 
 $plugin->version   = 2023120801;              // Current version of the plugin (Date: YYYYMMDDXX).
-$plugin->requires  = 2020061524;              // Requires Moodle 3.9 and above.
+$plugin->requires  = 2023042400;              // Requires Moodle 3.9 and above.
```

Comment:
Just one nitpick, the comment should be updated to say 4.2

Reply:
Thanks, good catch, this is resolved.